placetobe-campaign-page.html

```diff
@@ -505,12 +500,27 @@
         },
 
         incentives: {
-          type: Array
+          type: Array,
+          computed: "_computeAvailablesIncentives(_groupedIncentives.*, contractTypes.*)"
```

Comment:
`_groupedIncentives` is een object
`contractTypes` is een Array maar kreeg niet aan de praat met splices gek genoeg. 
Iets meer rekenen met * maar met zo'n kleine Array is dat verwaarloosbaar